assignment6/polls/constants.py

```diff
@@ -0,0 +1,17 @@
+# well known constants
+
+# data
+MOVIE_FILE = "C:/Users/Karol/Documents/Python Scripts/search&recomandation/assignment6/polls/data/movies.csv"
+RATINGS_SMALL_FILE = "C:/Users/Karol/Documents/Python Scripts/search&recomandation/assignment6/polls/data/ratings_small.csv"
+RATINGS_FILE = ""
+RECOMENDATION_SITE_TEMPLATE ="C:/Users/Karol/Documents/Python Scripts/search&recomandation/assignment6/polls/templates/assignment6/secondpage.html"
```

Comment:
is it necessary to use absolut paths here? If not, pls use relative path so that it doesn't have to be changed every time someone has to touch the file